convert_mean.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
It is a good practice to use the `if __name__ == "__main__":` block in the runnable script.